pingmachine-status

```diff
@@ -175,6 +152,40 @@ sub main {
     }
 }
 
+sub scan_orders_dir {
+    my ($orders_dir, $order_id_prefix, $orders, $orders_by_user, $any_ipv6) = @_;
+    opendir(my $open_orders_dir, $orders_dir) or die "ERROR: can't open $ORDERS_DIR: $!\n";
+    while(my $order_file = readdir($open_orders_dir)) {
+        next if $order_file eq '.' or $order_file eq '..';
+
+        my $order_id = $order_id_prefix ? "$order_id_prefix/$order_file" : $order_file;
+        my $order_path = "$orders_dir/$order_file";
+        if (-d $order_path) {
+            scan_orders_dir($order_path, $order_id, $orders, $orders_by_user, $any_ipv6);
+        }
+
+        next unless -f "$order_path";
+        my $order;
+        try { $order = LoadFile("$order_path"); }
+        catch { warn "WARNING: can't parse $order_path\n"; };
+        next unless defined $order and defined $order->{user};
```

Comment:
Perl Best Practice: never use `and`, `or`and `not` as they have an unexpectedly low precedence level.
And for such a long check I would not use `unless` and also not a post `if`/`unless`
```suggestion
        next unless $order;
        next unless defined $order->{user};
```